src/recipes/vscode/index.js

```diff
@@ -44,15 +75,15 @@ const run = async ({ config, repositoryRoot }) => {
     return
   }
 
+  if (!(await hasDenoVSCodeExt())) {
```

Comment:
Sounds good to me.  What is going on is that the tests are failing because VSCode is not installed on the github actions runners, and so the execa call errors out.  I suppose there is a scenario where a user could have a .vscode folder but not actually have it installed and it triggers the recipe to run?

Reply:
> I guess my question is, if they do not have VSCode installed, why should I still display a helpful message guiding them to the Deno Extension?

If this code is running, it means that either we detected that the terminal is VSCode's or the user typed the `vscode` recipe.

In both cases, there are high chances that they do have VSCode installed and the error is due to something else: the `code` binary is not in the shell path, we failed to parse the output, etc.

But even if they don't have it installed, pointing them to the extension feels more helpful than just saying "there was an error", because the URL they'll land on will eventually tell them that they need VSCode installed and offer them the option to do it.

Reply:
As for the tests, you can mock execa if needed. We do it in some tests (you can search for `execa` in `*.test.js` files).